config/config.go

```diff
@@ -908,186 +908,6 @@ func SetupViper(v *viper.Viper, filename string) {
 	// macro substitution. it is important for the uid to be the last query parameter.
 	v.SetDefault("user_sync.redirect_url", "{{.ExternalURL}}/setuid?bidder={{.SyncerKey}}&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&f={{.SyncType}}&uid={{.UserMacro}}")
 
-	for _, bidder := range openrtb_ext.CoreBidderNames() {
```

Comment:
without any defaults, how will viper know to look for adapter config overrides in the environment variables. Viper does not use reflect to interrogate the config struct for all possible fields that might exist. I do think viper has another function other than `SetDefault()` that will clue it in to a field's existence without actually setting a default value.

Reply:
This also relates to Configuration -> map[string]Adapter which was removed. There is no way to set adapterDefaults to adapters. Unless i understand something wrong. We need to discuss it with the team.

Reply:
See my comments below on environment variable config support being removed.